userspace/falco/falco_metrics.cpp

```diff
@@ -93,15 +93,15 @@ std::string falco_metrics::to_text(const falco::app::state& state)
 		{
 			fs::path fs_path = item.first;
 			std::string metric_name_file_sha256 = fs_path.filename().stem();
-			metric_name_file_sha256 = "falco.sha256_rules_file." + falco::utils::sanitize_metric_name(metric_name_file_sha256);
+			metric_name_file_sha256 = "falco_sha256_rules_file_" + falco::utils::sanitize_metric_name(metric_name_file_sha256);
```

Comment:
That was a super small regression I must have introduced in the last PR. We will push some better checks and sanitizing the metrics name into libs in the upcoming dev cycle, already tracking it in our issue.

Reply:
Strange, I thought I saw this dot that look out of place but somehow missed to ask about it...

I think we should add a more thorough test  of the output content to avoid such regression in the future.

Reply:
Agreed, plus even push more checks into libs for which we didn't have time anymore. It's tracked on that one issue we have.